docs/source/examples/checks/distribution/train_test_feature_drift.ipynb

```diff
@@ -3,26 +3,28 @@
   {
```

Comment:
<p>remove the #</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I don't where it is coming from here, it doesn't exists in the NB</p>

Reply:
seems like it's a bug in the reviewnb.